apps/drupal-default/particle_theme/templates/block/block--utc-image-hover-effect.html.twig

```diff
@@ -0,0 +1,106 @@
+{% extends "block.html.twig" %}
+{#
+/**
+ *
+ * Implements a grid of cards using Tailwinds layout.
+ *
+ * Available variables:
+ */
+ *
+ * This template pulls in the variables to add to hover image effects. 
+ * References the block--utc-image-hover-effect-base.html.twig to create the individual cards in this grid.
+#}
+
+{% block content %}
+	{% set rendered_content = content|render %}
+	{# Sets variable to trigger content render array. #}
+
+
+	{# I belive all of these variables defined by Drupal inputs would remain here, but the
+  grid portion would be moved to a different template file if/when a design system is implemented #}
+
+	{% set hover_count = content["#block_content"].field_hover_image_count.0.value %}
+	{% set hover_orientation = content["#block_content"].field_hover_image_orientation.0.value %}
+	
+	{% if hover_count == 1 %}
+		{% set single_hover = [
+			'hover-count-1',
+			'mb-8',
+			'md:mb-0',
+		]%}
+	{% elseif hover_count == 2 %}
+		{% set single_hover = [
+			'hover-count-2',
+			'mb-8',
+			'md:mb-0',
+		]%}
+	{% elseif hover_count == 3  %}
+		{% set single_hover = [
+			'hover-count-3',
+			'mb-8',
+			'md:mb-0',
+		]%}
+	{% endif %}
+
+	{%
+		set hover_classes = [
+			'utc-image-hover',
+			hover_orientation ? hover_orientation,
+		]
+	%}
+
+	{# set variables for card content based on Drupal input #}
+
+	{% set item_hover_1 = {
+		hover_image: content.field_hover_image_one|field_value,
```

Comment:
I think we might be able to cut down in the number of fields by doing what we did on the button group.
So there would be only one field_hover_image instead of field_hover_image_one field_hover_image_two and field_hover_image_three and we can access the values trough a for loop.

Basically we create a field_hover_image and then the second time we want to add a field we reuse that field which is an option right next to create.

Reply:
https://github.com/UTCWeb/particle/blob/develop/apps/drupal-default/particle_theme/templates/block/block--block-content--utc-button-group.html.twig

Reply:
I will look into this.

Reply:
@bmartinez287 is there a way to make entire field groups optionally able to be duplicated? For example, keeping the title, description, link, etc. together as a set and duplicate the entire set of fields each time as needed to add cards (or by the numbers items). Just thinking, if the image field can be duplicated and then the title field can be duplicated...but they aren't grouped logically...it might cause some user confusion when building?

Also, is this module relevant when trying to do something like this? https://www.drupal.org/project/business_rules

Reply:
@UTCbridget I forgot to reply. We are currently researching and prototyping with a couple of alternatives to simplify the twig templates. Bridget is looking into paragraph components and I'm looking into extending fields through custom modules. We should have something by tomorrow night that could simplify the UI on these components. This is currently the main priority we are working on as we want to get all your components in before November.

Reply:
@brittrich if you could prioritize improving the Figma files that be great this week. Mostly since new components will change slightly once we provide a new way for them to be grouped

Reply:
Sounds great and makes perfect sense! I'll spend my time this week in Figma.